test/integration/M3/Setup.t.sol

```diff
@@ -0,0 +1 @@
+// TODO
```

Comment:
is there any benefit of this file?

Reply:
https://github.com/yieldnest/yieldnest-protocol/pull/159/commits/43c4269365f7cfdee928ac5abff1fce94cabd04a